pkg/multiTenancyPlugins/quota/quota.go

```diff
@@ -80,11 +80,6 @@ func (quotaImpl *DefaultQuotaImpl) Handle(command string, cluster cluster.Cluste
 	case "containerdelete":

```

Comment:
Why did you remove the check? what happen if I delete through Swarm a container that created by K8 ? Are you going to decrease the quota ?